.changeset/new-gifts-thank.md

```diff
@@ -0,0 +1,7 @@
+---
+"@logto/core": patch
+"@logto/integration-tests": patch
+"@logto/schemas": patch
+---
+
+update webhook API and schema
```

Comment:
better to specify what has been updated. see [this file](https://github.com/logto-io/logto/pull/3792/files#diff-220c54ad6a33dc51c8944635aa12755d2dce7669cefbca172864940cd5f98af2)

Reply:
Cool, updated.

Reply:
actually you can write the changeset from a feature perspective, i.e. a changeset can be the results of several PRs. in this case, you can even add the changeset in the next PR to describe the whole picture of the hook update

Reply:
Yes, a new changeset will be added when the whole feature is implemented.